src/ForceIDPDiscoveryController.js

```diff
@@ -0,0 +1,57 @@
+/*!
+ * Copyright (c) 2015-2017, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import Spinner from 'views/shared/Spinner';
+import BaseLoginController from 'util/BaseLoginController';
+import Util from 'util/Util';
+
+export default BaseLoginController.extend({
+  className: 'force-idp-discovery',
+
+  View: Spinner,
+
+  Model: {},
+
+  initialize: function() {
+    let options = this.options;
+    const lastAuthResponse = options.appState.get('lastAuthResponse');
+    const stateToken = lastAuthResponse && lastAuthResponse?.stateToken;
+
+    const webfingerArgs = {
+      resource: 'okta:acct:undefined',
```

Comment:
nit:
just to confirm `undefined` means there is no user name right? is `resource` a required parameter? may be we just don't pass it? 
I mean I am okay with special case string, just that `undefined` sounds weird as an input since its special keyword in JS.

Reply:
I use this because it's the same value as you'd submit if you didn't type in a username in the regular idp discovery flow (from the model):
```
    const username = this.settings.transformUsername(this.get('username'), Enums.IDP_DISCOVERY);
    const remember = this.get('remember');
    const lastUsername = this.get('lastUsername');
    const resource = 'okta:acct:' + username;
 ```

It is a required parameter on the backend, so we do need to send something.

After doing some investigation on the backend, it basically treats this as a user that has the username `undefined`. I think we should instead send a blank username (which is always invalid) as opposed to the `undefined` string, that way it won't match with any users which is what we want in this particular case.